src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1355,6 +1541,11 @@ private static unsafe long ReadCompareExchangeIndex(TableValueReader reader)
             return *(long*)reader.Read((int)UniqueItems.Index, out var _);
         }
 
+        private static unsafe long ReadIdentitiesIndex(TableValueReader reader)
+        {
+            return *(long*)reader.Read((int)UniqueIdentitiesItems.Index, out var _);
```

Comment:
Add a debug assert here that the size is correct

Reply:
done